src/current/releases/molt.md

```diff
@@ -18,6 +18,18 @@ To download the latest MOLT Fetch/Verify binary:
 
 {% include molt/molt-install.md %}
 
+## December 13, 2024
+
+MOLT Fetch/Verify 1.2.1 is [available](#installation).
+
+- MOLT Fetch users now can use [`--assume-role`]({% link molt/molt-fetch.md %}#global-flags) to specify a service account for assume role authentication to cloud storage. `--assume-role` must be used with `--use-implicit-auth`, or it will be ignored. 
```

Comment:
```suggestion
- MOLT Fetch users now can use [`--assume-role`]({% link molt/molt-fetch.md %}#global-flags) to specify a service account for assume role authentication to cloud storage. `--assume-role` must be used with `--use-implicit-auth`, or the service account will be ignored. 
```

What do you think? I had to do a quick re-read to realize that "it" was referring to assume role.

Reply:
Hmm I think we can say "the flag will be ignored" as that's simplest.